playbooks/installation/install_validation.adoc

```diff
@@ -109,16 +111,20 @@ Follow the steps below
 [Wait for build to start]
 # oc get pods | grep build
 [ copy the pod id ]
-# oc logs <pod id>
+# oc logs -f <pod id>
 [ Look for message saying an image was successfully pushed ]
-# oc get deployments
-[ make sure one has succeeded ]
+# oc get deploymentConfig
+[ copy the deploymentConfig name ]
+# oc deploy <deploymentConfig>
```

Comment:
@JaredBurck that's exactly what I'm trying to do here.. view the latest deployment to ensure it succeeded.